### PR TITLE
explorer: get vote plan info by id

### PR DESCRIPTION
**Pull request by eugene-babichenko**
TODO:

- [x] ~Add account values~
- [x] Implement vote tally (depends on account values)
- [x] See what fields are missing and add them
- [x] Add fetching multiple vote plans at once (paginated)
- [x] Have some clue about reward distribution

### CODE REVIEW

File: jormungandr/src/explorer/indexing.rs [hunk 1]
```diff
@@ -89,6 +94,30 @@ pub enum ExplorerAddress {
...
+    pub payload_type: PayloadType,
+    pub proposals: Vec<ExplorerVoteProposal>,
+}
+
+#[derive(Clone)]
+pub struct ExplorerVoteProposal {
+    pub proposal_id: ExternalProposalId,
+    pub options: Options,
+    pub tally: Option<ExplorerVoteTally>,
+    pub votes: std::collections::HashMap<ExplorerAddress, u8>,
```
Comment (NicolasDP) [inline jormungandr/src/explorer/indexing.rs:112]: what is this `u8` ?

Comment (eugene-babichenko) [reply jormungandr/src/explorer/indexing.rs:112]: Huh, you are right. This should be `Choice`.

File: jormungandr/src/explorer/graphql/scalars.rs [hunk 2]
```diff
@@ -51,6 +51,12 @@ pub enum PayloadType {
     Public,
 }
 
+#[derive(juniper::GraphQLScalarValue)]
+pub struct Address(pub String);
```
Comment (enzoc4) [inline jormungandr/src/explorer/graphql/scalars.rs:55]: why do we need a new scalar type and not the existing Address type?

File: jormungandr/src/explorer/indexing.rs [hunk 3]
```diff
@@ -89,6 +94,33 @@ pub enum ExplorerAddress {
...
+    pub payload_type: PayloadType,
+    pub proposals: Vec<ExplorerVoteProposal>,
+}
+
+#[derive(Clone)]
+pub struct ExplorerVoteProposal {
+    pub proposal_id: ExternalProposalId,
+    pub options: Options,
+    pub tally: Option<ExplorerVoteTally>,
+    pub votes: std::collections::HashMap<ExplorerAddress, Choice>,
```
Comment (enzoc4) [inline jormungandr/src/explorer/indexing.rs:112]: I'm curious about this `HashMap` . Is it not another `Hamt` for a reason? Is it fine to clone it on every vote?

**Pull request merged**